ext/openssl/openssl.stub.php

```diff
@@ -103,12 +103,12 @@ function openssl_get_privatekey($private_key, ?string $passphrase = null): OpenS
 
 function openssl_pkey_get_details(OpenSSLAsymmetricKey $key): array|false {}
 
-function openssl_pbkdf2(string $passphrase, string $salt, int $key_length, int $iterations, string $digest_algorithm = "sha1"): string|false {}
+function openssl_pbkdf2(string $passphrase, string $salt, int $key_length, int $iterations, string $digest_algo = "sha1"): string|false {}
```

Comment:
Relative to hash_pbkdf2 we have an inconsistency here with passphrase vs password and key_length vs length. Though I guess it's more important to be internally consistent inside openssl here.

Reply:
I'm ok to use `$passphrase` in ext/hash as well. Or would you prefer it the other way around?

Reply:
At least where PBKDF2 (aka password-based key derivation function) is concerned, using `$password` would be more natural.